action.yml

```diff
@@ -65,4 +65,4 @@ runs:
     - ${{ inputs.dry_run }}
     - ${{ inputs.working_directory }}
     - ${{ inputs.initial_source_folder }}
-    - ${{ inputs.initial_commit_message }}
```

Comment:
GitHub's editor likes to add the missing newline at eof. If requested, I could make a local commit w/o this fix, but generally it's best practice to include that character.